src/patternfly/components/Sequence/sequence.scss

```diff
@@ -0,0 +1,374 @@
+
+  // Sequence Control
+  --pf-c-sequence__control--PaddingTop: var(--pf-global--spacer--xl);
+  --pf-c-sequence__control--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-sequence__control--PaddingBottom: var(--pf-global--spacer--xl);
+  --pf-c-sequence__control--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-sequence__control--BorderTopHeight: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--BorderBottomHeight: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--BorderTopColor: transparent;
+  --pf-c-sequence__control--BorderBottomColor: transparent;
+  --pf-c-sequence__control--hover--BorderTopHeight: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--active--BorderTopHeight: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--focus--BorderTopHeight: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--hover--BorderBottomHeight: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--active--BorderTopHeight: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--focus--BorderBottomHeight: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--hover--BorderTopColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__control--active--BorderTopColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__control--focus--BorderTopColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__control--hover--BorderBottomColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__control--active--BorderTopColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__control--focus--BorderBottomColor: var(--pf-global--primary-color--100);
+
+  // Sequence Control Left
+  --pf-c-sequence__control-left--PositionLeft: 0;
+  --pf-c-sequence__control-left--BackgroundColor: var(--pf-global--BackgroundColor--dark-100);
+  --pf-c-sequence__control-left--BorderRightWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-sequence__control-left--BorderRightColor: var(--pf-global--Color--200);
+
+  // Sequence Control Right
+  --pf-c-sequence__control-right--PositionRight: 0;
+  --pf-c-sequence__control-right--BackgroundColor: var(--pf-global--BackgroundColor--dark-100);
+  --pf-c-sequence__control-right--BorderLeftWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-sequence__control-right--BorderLeftColor: var(--pf-global--Color--200);
+
+  // Sequence Control Icon
+  --pf-c-sequence__control-icon--Color: var(--pf-global--Color--light-100);
+
+  // Item
+  --pf-c-sequence__item--MarginRight: 1px;
+  --pf-c-sequence__item--MarginLeft: 1px;
+  --pf-c-sequence__item--MaxWidth: 200px;
+
+  // Link
+  --pf-c-sequence__link--PaddingTop: var(--pf-global--spacer--xl);
+  --pf-c-sequence__link--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-sequence__link--PaddingBottom: var(--pf-global--spacer--lg);
+  --pf-c-sequence__link--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-c-sequence__link--BorderBottomColor: var(--pf-global--Color--200);
```

Comment:
Per the design, this should be semi transparent.

Reply:
Turns out, when I change the colors to work for both light/dark themes that these colors actually become white, to pass accessibility tests, so im not sure if transparency will be in the picture